### PR TITLE
Fix/Avellaneda - remove _latest_parameter_calculation_vol

### PR DESCRIPTION
**Pull request by mhrvth**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

_latest_parameter_calculation_vol parameter needs to be removed. It's a leftover of a previous implementation, it's use corrupts volatility measurement.

**Tests performed by the developer**:

- ran the strategy with ascend_ex paper
- ran unit test

### CODE REVIEW

**Comment by mhrvth** [comment]
[sc-23664]

File: hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx [hunk 1]
```diff
@@ -137,7 +137,6 @@ cdef class AvellanedaMarketMakingStrategy(StrategyBase):
         self._start_time = start_time
         self._end_time = end_time
         self._min_spread = min_spread
-        self._latest_parameter_calculation_vol = s_decimal_zero
```
Comment (aarmoa) [inline hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx:140]: If we remove this variable from the .pyx file we need to remove it also from the .pxd file

**Review by aarmoa — CHANGES_REQUESTED** [review]

**Review by RC-13 — APPROVED** [review]
LGTM!

Test Summary:
    * Create and started an Avellaneda MM strategy using paper trade connector and live connector and confirmed that no issue occurred during the initialization of the order books.
    * Configure different parameters to make sure they did not break
    * Check the volatility in the status window and confirmed that it adjust from time to time
    * Confirmed that trades are recorded and confirmed that orders are adjusted and behaved according to the specified parameters.

**Review by aarmoa — APPROVED** [review]
All changes look good to me

**Comment by mhrvth** [comment]
Created a PR in the Foundation repo: https://github.com/hummingbot/hummingbot/pull/5146

**Pull request closed without merging**